config/setup.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/edaniels/golog"
+	"github.com/pkg/errors"
+	"go.opencensus.io/trace"
+	pb "go.viam.com/api/service/slam/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// This increases the message size from 4MB to 32MB to match the RDK.
+// This is necessary for transmitting large pointclouds.
+var grpcMaxMessageSize = 32 * 1024 * 1024
+
+// SetupDirectories creates the data directory at the specified path along with
+// its data, map, and config subdirectories.
+func SetupDirectories(dataDirectory string, logger golog.Logger) error {
```

Comment:
Should we also set up the rgb and depth directories, or will that be done in the orbslam module?

Reply:
In my opinion that seems best to be left to the orbslam module as it is implementation-specific and adding those might tie the slam repo to single implementations. However, we could alter this function to accept an array of subdirectories to create. Deferring to @kkufieta

Reply:
I agree with @zaporter-work. The original plan was indeed to have this part in the orbslam repo: https://github.com/viamrobotics/rdk/blob/3089e392730f48fe5b5c7133258f6a26679f0c56/services/slam/builtin/builtin.go#L127-L143